last  git commit  failed to build in arch

### Already reported ? *

- [X] I have searched the existing open and closed issues.

### Regression?

No

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
hyprctl systeminfo -c
Hyprland, built from branch main at commit d73c14751ad40fd54d93baac2226f550142b0e74  (CI/Nix: git+https -> github).
Date: Sat Sep 28 13:53:18 2024
Tag: v0.43.0-79-gd73c1475, commits: 5275
built against aquamarine 0.4.1


flags: (if any)


System Information:
System name: Linux
Node name: rocket
Release: 6.10.10-arch1-1
Version: #1 SMP PREEMPT_DYNAMIC Thu, 12 Sep 2024 17:21:02 +0000


GPU information: 
00:02.0 VGA compatible controller [0300]: Intel Corporation TigerLake-LP GT2 [Iris Xe Graphics] [8086:9a49] (rev 01) (prog-if 00 [VGA controller])


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:

======Config-Start======


Config File: /home/alek/.config/hypr/hyprland.conf: Read Succeeded

# For a full list, see the wiki
#

#autogenerated = 1 # remove this line to remove the warning

# See https://wiki.hyprland.org/Configuring/Monitors/
#monitor=,preferred,auto,1
monitor=HDMI-A-1, 3840x2160@60, 0x0, 1
monitor=eDP-1, 1920x1080@60, 3840x0, 1

#workspace workspace=HDMI-A-1,1
workspace=1, monitor:HDMI-A-1
workspace=2, monitor:HDMI-A-1
workspace=3, monitor:HDMI-A-1
workspace=4, monitor:HDMI-A-1
workspace=5, monitor:HDMI-A-1
workspace=6, monitor:HDMI-A-1
workspace=7, monitor:HDMI-A-1
workspace=8, monitor:eDP-1
workspace=9, monitor:eDP-1
# See https://wiki.hyprland.org/Configuring/Keywords/ for more

# Execute your favorite apps at launch
# exec-once = waybar & hyprpaper & firefox

#exec-once = waypaper --restore
exec-once = swaybg -i /usr/share/hypr/wall2.png
exec-once = waybar
#exec-once = [workspace 6 silent] pypr
exec-once = pypr --debug /tmp/pypr.log
#exec-once = dunst
exec-once = wl-paste --type text --watch cliphist store #Stores only text data
exec-once = wl-paste --type image --watch cliphist store #Stores only image data

#portal
exec-once = portal.sh

# Source a file (multi-file configs)
# source = ~/.config/hypr/myColors.conf

# Set programs that you use
$terminal = kitty
$fileManager = ranger
$menu = tofi-drun --drun-launch=true 
$firefox = firefox
$librewolf = librewolf
$brave     = brave
$chromium = chromium 
$chat	 = weechat
$nvim     = nvim
$graphics 	 = gimp
$graphics2	 = blender
$screenshot	 = grim
$nwg-menu	 = nwg-bar
$paste		 = wl-paste
#$disable_mouse = /usr/local/bin/mouse_toggle.sh; sleep 10
# Some default env vars.
env = XCURSOR_SIZE,24
env = QT_QPA_PLATFORMTHEME,qt5ct # change to qt6ct if you have that
env = HYPRCURSOR_THEME,rose-pine-hyprcursor

# For all categories, see https://wiki.hyprland.org/Configuring/Variables/
input {
    kb_layout = us
    kb_variant =
    kb_model =
    kb_options = caps:escape
    kb_rules =

    follow_mouse = 0
    #scroll_method = no_scroll
    touchpad {
        natural_scroll = no
	disable_while_typing = true
    }

    sensitivity = 0 # -1.0 - 1.0, 0 means no modification.
    float_switch_override_focus = 0
    
    scroll_method = no_scroll 
}

general {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more

    gaps_in = 5
    gaps_out = 30
    border_size = 4
    col.active_border = rgba(33ccffee) rgba(00ff99ee) 45deg
    col.inactive_border = rgba(595959aa)

    layout = dwindle

    # Please see https://wiki.hyprland.org/Configuring/Tearing/ before you turn this on
    allow_tearing = false
    #no_cursor_warps = true
}
decoration {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more

    rounding = 15
    
    blur {
        enabled = true
        size = 20 
        passes = 1
	xray = true
    }

    drop_shadow = yes
    shadow_range = 4 
    shadow_render_power = 3
    col.shadow = rgba(1a1a1aee)
    layerrule = blur, notifications
    layerrule = ignorezero, notifications
}

animations {
    enabled = yes

    # Some default animations, see https://wiki.hyprland.org/Configuring/Animations/ for more

    bezier = myBezier, 0.05, 0.9, 0.1, 1.05

    animation = windows, 1, 7, myBezier
    animation = windowsOut, 1, 7, default, popin 80%
    animation = border, 1, 10, default
    animation = borderangle, 1, 8, default
    animation = fade, 1, 7, default
    animation = workspaces, 1, 6, default
}

dwindle {
    # See https://wiki.hyprland.org/Configuring/Dwindle-Layout/ for more
    pseudotile = yes # master switch for pseudotiling. Enabling is bound to mainMod + P in the keybinds section below
    preserve_split = yes # you probably want this
}

master {
    # See https://wiki.hyprland.org/Configuring/Master-Layout/ for more
    new_status = master
}

gestures {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more
    workspace_swipe = off
}

misc {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more
    force_default_wallpaper = 0 # Set to 0 to disable the anime mascot wallpapers
    disable_hyprland_logo = true
    enable_swallow = false
}

# Example per-device config
# See https://wiki.hyprland.org/Configuring/Keywords/#executing for more
 device {
    name = epic-mouse-v1
    sensitivity = -0.5
}

# Example windowrule v1
# windowrule = float, ^(kitty)$
# Example windowrule v2
# windowrulev2 = float,class:^(kitty)$,title:^(kitty)$
# See https://wiki.hyprland.org/Configuring/Window-Rules/ for more
windowrulev2 = float, class:^(Alacritty)$
windowrulev2 = center[1], class:^(Alacritty)$
windowrulev2 = pin, class:^(Alacritty)$
windowrulev2 = stayfocused, class:^(Alacritty)$

windowrulev2 = float,class:^(pavucontrol)$
windowrulev2 = center[1], class:^(Alacritty)$
windowrulev2 = pin,class:^(pavucontrol)$
windowrulev2 = stayfocused,class:^(pavucontrol)$
windowrulev2 = size 30% 40%,class:^(pavucontrol)$
#windowrulev2 = float, class:^(neomutt)$
#windowrulev2 = workspace 7, class:neomutt1


#browsers
#windowrulev2 = float,class:^(Chromium)$
windowrulev2 = float,class:^(org.qutebrowser.qutebrowser)$

#graphics
windowrulev2 = workspace 8, class:^(Gimp-2.10)$
windowrulev2 = workspace 8, class:^(blender)$


#librewolf
windowrulev2 = workspace 7, class:^(LibreWolf)$

#zathura 
windowrulev2 = float,class:^(org.pwmt.zathura)$
windowrulev2 = size 60% 60%, class:^(org.pwmt.zathura)$


#default
windowrulev2 = suppressevent maximize, class:.* # You'll probably like this.

# See https://wiki.hyprland.org/Configuring/Keywords/ for more
$mainMod  = SUPER
$mainMod4 = CTRL
$mainMod2 = ALT
$mainMod24 = ALT+CTRL

#common commands
bind = $mainMod, C, killactive, 
bind = $mainMod SHIFT, E, exit, 
bind = $mainMod, Space, togglefloating, 
bind = $mainMod, D, exec, $menu
bind = $mainMod, P, pseudo, # dwindle
bind = $mainMod SHIFT, P, exec, $nwg-menu
bind = $mainMod, S, togglesplit, # dwindle
bind = $mainMod, F, exec, hyprctl dispatch fullscreen 

# my apps
bind = $mainMod, Return, exec, $terminal
bind = $mainMod, A, exec, $brave --ozone-platform-hint=auto
bind = $mainMod, W, exec, $terminal $chat

#untilities
bind = $mainMod, F1, exec, $screenshot -g "$(slurp)" - | wl-copy #copy the screenshot
bind = $mainMod, F2, exec,  cliphist list | tofi | cliphist decode | wl-copy #start tofi with cliphist to choose from 
bind = $mainMod2, h,  exec,  ~/commands/hyperland/reload.sh 
bind = $mainMod2, w,  exec,  killall -SIGUSR2 waybar
bind = $mainMod,  l,  exec,  hyprlock
bind = $mainMod SHIFT, e,  exec,  hyprctl dispatch exit
bind = $mainMod SHIFT, r,  exec,  systemctl reboot
bind = $mainMod SHIFT, s,  exec,  systemctl -i poweroff 

#cycle through the list of windows in the workflow
bind =  $mainMod, Tab, cyclenext

# Switch workspaces with mainMod + [0-9]
bind = $mainMod, 1, workspace, 1
bind = $mainMod, 2, workspace, 2
bind = $mainMod, 3, workspace, 3
bind = $mainMod, 4, workspace, 4
bind = $mainMod, 5, workspace, 5
bind = $mainMod, 6, workspace, 6
bind = $mainMod, 7, workspace, 7
bind = $mainMod, 8, workspace, 8
bind = $mainMod, 9, workspace, 9

# Move active window to a workspace with mainMod + SHIFT + [0-9]
bind = $mainMod SHIFT, 1, movetoworkspace, 1
bind = $mainMod SHIFT, 2, movetoworkspace, 2
bind = $mainMod SHIFT, 3, movetoworkspace, 3
bind = $mainMod SHIFT, 4, movetoworkspace, 4
bind = $mainMod SHIFT, 5, movetoworkspace, 5
bind = $mainMod SHIFT, 6, movetoworkspace, 6
bind = $mainMod SHIFT, 7, movetoworkspace, 7
bind = $mainMod SHIFT, 8, movetoworkspace, 8
bind = $mainMod SHIFT, 9, movetoworkspace, 9
bind = $mainMod SHIFT, 0, movetoworkspace, 10

# Example special workspace (scratchpad)
#bind = $mainMod, S, togglespecialworkspace, magic
#bind = $mainMod SHIFT, S, movetoworkspace, special:magic

#pypr scratchpad
bind = $mainMod , N, exec, pypr toggle term
bind = $mainMod,  M, exec, pypr toggle neomutt1
bind = $mainMod,  B, exec, pypr toggle ncmpcpp
bind = $mainMod,  v, exec, pypr toggle volume
bind = $mainMod,  z, exec, pypr zoom
bind = $mainMod,  E, exec, pypr toggle ranger

# Scroll through existing workspaces with mainMod + scroll
#bind = $mainMod, mouse_down, workspace, e+1
#bind = $mainMod, mouse_up, workspace, e-1

# Move/resize windows with mainMod + LMB/RMB and dragging
bindm = $mainMod, mouse:272, movewindow
bindm = $mainMod, mouse:273, resizewindow

======Config-End========
```

</details>

### Description

Found ninja-1.12.1 at /usr/bin/ninja
INFO: autodetecting backend as ninja                                                                                                                                                                             
INFO: calculating backend command to run: /usr/bin/ninja -C /home/alek/.cache/paru/clone/hyprland-git/src/Hyprland/build
ninja: Entering directory `/home/alek/.cache/paru/clone/hyprland-git/src/Hyprland/build'
[201/292] Compiling C++ object src/Hyprland.p/managers_ProtocolManager.cpp.o
FAILED: src/Hyprland.p/managers_ProtocolManager.cpp.o 
c++ -Isrc/Hyprland.p -Isrc -I../src -Isubprojects/udis86 -I../subprojects/udis86 -I../subprojects/udis86/libudis86 -Isubprojects/udis86/__CMake_build -I../subprojects/udis86/__CMake_build -Iprotocols -I/usr/include/cairo -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/pango-1.0 -I/usr/include/fribidi -I/usr/include/uuid -flto=auto -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -fpch-preprocess -include pch.hpp -D_FILE_OFFSET_BITS=64 -std=c++26 -O3 -Wno-unused-parameter -Wno-unused-value -Wno-missing-field-initializers -Wno-narrowing -Wno-pointer-arith '-DDATAROOTDIR="/usr/share"' -DHAS_EXECINFO '-DAQUAMARINE_VERSION="0.4.1"' -DUSES_SYSTEMD -w -fPIE -pthread -MD -MQ src/Hyprland.p/managers_ProtocolManager.cpp.o -MF src/Hyprland.p/managers_ProtocolManager.cpp.o.d -o src/Hyprland.p/managers_ProtocolManager.cpp.o -c ../src/managers/ProtocolManager.cpp
In file included from ../src/managers/ProtocolManager.cpp:47:
../src/managers/../protocols/SinglePixel.hpp:7:10: fatal error: single-pixel-buffer-v1.hpp: No such file or directory
    7 | #include "single-pixel-buffer-v1.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[210/292] Compiling C++ object src/Hyprland.p/managers_PointerManager.cpp.o
ninja: build stopped: subcommand failed.
==> ERROR: A failure occurred in build().
    Aborting...
error: failed to build 'hyprland-git-0.42.0.r64.82c67e61-1': 
error: packages failed to build: hyprland-git-0.42.0.r64.82c67e61-1

### How to reproduce

try to build using aur hyprland-git 

### Crash reports, logs, images, videos

_No response_

--- a/protocols/meson.build
+++ b/protocols/meson.build
@@ -63,6 +63,7 @@ protocols = [
 	wayland_protocol_dir / 'staging/drm-lease/drm-lease-v1.xml',
 	wayland_protocol_dir / 'staging/linux-drm-syncobj/linux-drm-syncobj-v1.xml',
 	wayland_protocol_dir / 'staging/xdg-dialog/xdg-dialog-v1.xml',
+	wayland_protocol_dir / 'staging/single-pixel-buffer/single-pixel-buffer-v1.xml',
 ]
 
 wl_protocols = []

--- a/src/protocols/SinglePixel.cpp
+++ b/src/protocols/SinglePixel.cpp
@@ -1,5 +1,6 @@
 #include "SinglePixel.hpp"
 #include <limits>
+#include "render/Renderer.hpp"
 
 CSinglePixelBuffer::CSinglePixelBuffer(uint32_t id, wl_client* client, CColor col_) {
     LOGM(LOG, "New single-pixel buffer with color 0x{:x}", col_.getAsHex());

